pyproject.toml

```diff
@@ -31,6 +31,7 @@ aiozmq = "0.7.1"
 
 [tool.poetry.dev-dependencies]
 asynctest = "0.12.2"
-pytest = "4.1.0"
-pytest-cov = "2.6.1"
+pytest = "4.0.1"
```

Comment:
Why did you downgrade those two packages?

Reply:
@eugene-babichenko, I didn't. Just while merging a conflict I add one dependency and moved few old below this new. So git considered this as deleted 2 and added 3 or something like that. There is still all packages plus pytest-asyncio for testing async function inside tests methods (to test async function, tests should be also async).